backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -180,7 +181,11 @@ public int getHeight () {
 		return height;
 	}
 
-	public Buffer getPixels () {
+	public ByteBuffer getPixels () {
+		if (pixels == null) pixels = context.getImageData(0, 0, width, height).getData();

```

Comment:
If you update the pixmap (via one of the drawing methods) does this buffer get updated?